actions/websocket_actions.jsx

```diff
@@ -708,21 +708,20 @@ export async function handleUserRemovedEvent(msg) {
             if (msg.data.remover_id === msg.broadcast.user_id) {
                 browserHistory.push(getCurrentRelativeTeamUrl(state));
             } else {
-                let user = getUser(state, msg.data.remover_id);
+                const user = getUser(state, msg.data.remover_id) || {};
                 if (!user) {
-                    await dispatch(loadUser(msg.data.remover_id));
-                    user = getUser(state, msg.data.remover_id) || {};
+                    dispatch(loadUser(msg.data.remover_id));
```

Comment:
this code is not reachable

Reply:
actually it is, if the user that removed you is not loaded yet, normally related to guest accounts loading only certain data. for example because is not in the current team. (A system admin can remove you from a channel without been member of the chanel itself).

Reply:
my bad, you are right, It shoudn't be `|| {}`, fixing it.